### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by takshch**
Added Bullets, Heading , Highlight css files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
-# website
+# Website
```
Comment (ankushdharkar) [inline README.md:1]: Why jump from `#` to `###`

Always have linear increments, so make it an h2 please (`##`)

File: README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-
---> global css
-    --> main.css - this css will contain  common css pertaining to all pages that can be used throughout the website. for e.g. a class
-    with name success, when used anywhere throughout the website will make the color of the text green.
---> page specific css 
-    for example, events page css goes inside events.css 
---> also, in future if we add new components, we will keep component specific css like navbar.css, footer.css, header.css etc
---> for mobile responsive css, we can have a folder with multiple css files catering to different css pertaining to different resolutions 
+### CSS Structure    
+#### Global CSS
```
Comment (ankushdharkar) [inline README.md:6]: This will then become ###

File: README.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
-    --> main.css - this css will contain  common css pertaining to all pages that can be used throughout the website. for e.g. a class
-    with name success, when used anywhere throughout the website will make the color of the text green.
---> page specific css 
-    for example, events page css goes inside events.css 
---> also, in future if we add new components, we will keep component specific css like navbar.css, footer.css, header.css etc
---> for mobile responsive css, we can have a folder with multiple css files catering to different css pertaining to different resolutions 
+### CSS Structure    
+#### Global CSS  
+   * `main.css`
+      * This css file will contain common css pertaining to all pages that can be used throughout the website.
```
Comment (ankushdharkar) [inline README.md:8]: Let's use `-` instead of `*`

File: README.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
---> page specific css 
-    for example, events page css goes inside events.css 
---> also, in future if we add new components, we will keep component specific css like navbar.css, footer.css, header.css etc
---> for mobile responsive css, we can have a folder with multiple css files catering to different css pertaining to different resolutions 
+### CSS Structure    
+#### Global CSS  
+   * `main.css`
+      * This css file will contain common css pertaining to all pages that can be used throughout the website.   
+      * For example, a class with name success, when used anywhere throughout the website will make the color of the text green.  
+#### Page specific CSS
```
Comment (ankushdharkar) [inline README.md:10]: Same

File: README.md (hunk 1 above)
Comment (ankushdharkar) [reply README.md:1]: Website is h1, CSS Structure should be h2

Comment (takshch) [reply README.md:1]: Okay. Done!

File: README.md (hunk 4 above)
Comment (takshch) [reply README.md]: All changes are made. Please review again.

**Pull request merged**